modules/oe_theme_helper/tests/src/Kernel/Plugin/FieldFormatter/AddressInlineFormatterTest.php

```diff
@@ -31,7 +31,7 @@ protected function setUp(): void {
    * Tests formatting of address.
    */
   public function testInlineFormatterAddress() {
-    $entity = EntityTest::create([]);
+    $entity = EntityTestMul::create([]);
```

Comment:
Any reason why we changed the entity type here? We don't use anything of `EntityTestMul` in the test.

Reply:
EntityTest - it is not translatable entity type
But EntityTestMul is translatable.
I have found that not translatable entity type does not pass to formatter class a langcode (passing null).

Reply:
Because we still use \Drupal\Tests\address\Kernel\Formatter\FormatterTestBase as parent class. But it is changed in patch of contribs module.
I did this for have possibility to tests address fields with translatable entity types.
EntityTest - do not allow to do this.

Reply:
Ok but here, in this test, we are not testing its translatablility right? That's why I was thinking to just use EntityTest

Reply:
Ah I see it, it's because of `$this->renderEntityFields($entity, $this->display);`. Resolving.